includes/base/controls-stack.php

```diff
@@ -444,7 +431,21 @@ public function add_control( $id, array $args, $options = [] ) {
 			}
 		}
 
-		if ( $should_optimize_controls ) {
+		if ( $this->should_optimize_controls() ) {
+			$ui_controls = [
+				Controls_Manager::RAW_HTML,
+				Controls_Manager::DIVIDER,
+				Controls_Manager::HEADING,
+				Controls_Manager::BUTTON,
```

Comment:
there is a new alert UI control I think, ask @rami-elementor

Reply:
Yes, two new UI controls. And I see you are missing another existing control